### PR TITLE
[systemtest] OLM - fix for wrong operator group and missing pull secret

### PR DESCRIPTION
**Pull request by im-konge**
### Type of change

- Bugfix
 
### Description

This PR fixes two issues:

- wrong operator group when we want to deploy CO via OLM that will watch multiple namespaces
- missing pull-secret when we want to pull images from some private registries

### Checklist

- [ ] Make sure all tests pass

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -314,6 +314,11 @@
                 <version>${fabric8.kubernetes-client.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>io.fabric8</groupId>
+                <artifactId>openshift-model-operatorhub</artifactId>
+                <version>${fabric8.openshift-client.version}</version>
+            </dependency>
```
Comment (scholzj) [inline pom.xml:321]: Should it be test scope dependency?

File: systemtest/src/test/java/io/strimzi/systemtest/upgrade/OlmUpgradeST.java [hunk 2]
```diff
@@ -77,6 +78,8 @@ void testStrimziUpgrade(ExtensionContext extensionContext) throws IOException {
         procedures.put("logMessageVersion", testKafkaVersion.messageVersion());
         procedures.put("interBrokerProtocolVersion", testKafkaVersion.protocolVersion());
         latestUpgradeData.put("proceduresAfterOperatorUpgrade", procedures);
+        // This is needed to overwrite for 2.0.0 OLM
+        latestUpgradeData.put("fromVersion", "1.8.0");
```
Comment (scholzj) [inline systemtest/src/test/java/io/strimzi/systemtest/upgrade/OlmUpgradeST.java:82]: There is no 1.8.0 / 2.0.0 => should this be parametrized?

Comment (im-konge) [reply systemtest/src/test/java/io/strimzi/systemtest/upgrade/OlmUpgradeST.java:82]: That's just some leftovers, gonna remove it, thanks

File: pom.xml (hunk 1 above)
Comment (im-konge) [reply pom.xml:321]: It's not directly in `test` scope, so I can't add it there.

**Review by scholzj — APPROVED** [review]

**Pull request merged**